sources/HUBComponentWrapper.h

```diff
@@ -167,6 +167,9 @@ willUpdateSelectionState:(HUBComponentSelectionState)selectionState;
 /// Whether the wrapped component's view has appeared since the model was last changed
 @property (nonatomic, readonly) BOOL viewHasAppearedSinceLastModelChange;
 
+/// The wrapped component's appearance count
```

Comment:
Maybe explain a bit more what this actually does? So it's not just simply re-stating the property name 😅 How about:
```
/**
 *    The number of times the wrapped component has appeared on the screen
 *
 *    Incremented every time the component gets sent the -viewWillAppear message.
 */
```